etc/ProdOfflineConfiguration.py

```diff
@@ -222,10 +205,7 @@
            sizePerEvent = 4000,
            scenario = ppScenario)
 
-datasets = [ "Commissioning1_0T", "Commissioning2_0T", "Commissioning3_0T", "Commissioning4_0T" ]
-
-for dataset in datasets:
-    addDataset(tier0Config, dataset,
+    addDataset(tier0Config, dataset+"_0T",
```

Comment:
Yeah, I can do it. It was just taking a while, so I only did it to the datasets I was already restructuring for other reasons.